app/schemas/anonymous_users.py

```diff
@@ -0,0 +1,14 @@
+from json import load
+from marshmallow import Schema, fields,validate
+
+class AnonymousUsersSchema(Schema):
+
+    id = fields.UUID(dump_only=True)
+    email = fields.String(required=True, load_only=True)
```

Comment:
Please remove `load_only` from the **email** because there is no way how Frontend will render the user given that the email field is not returned
```suggestion
    email = fields.String(required=True)
```

Reply:
Worked on it.